Common Programming Concepts/Variables/Converting Type/task.md

```diff
@@ -0,0 +1,15 @@
+## Converting Type
+
+Make me compile!
+
+<div class="hint">
+  In variables3 we already learned how to make an immutable variable mutable
```

Comment:
We can change the variables3 task name to one we have - Mutable Variable Binding